README.md

```diff
@@ -1,114 +1,45 @@
-If `No`, on your first visit to your Grav instance Grav will propose you to create a new user.
-
-<hr/>
+Grav is a modern open source flat-file CMS.
 
-# [FR] Grav pour YunoHost
+> *This package allow you to install Grav quickly and simply on a YunoHost server.  
+If you don't have YunoHost, please see [here](https://yunohost.org/#/install) to know how to install and enjoy it.*
 
 * http://getgrav.org/
 * https://github.com/getgrav/grav
 
-## Installation
-
-Depuis le panneau d'administration YunoHost:
+Provided version: **1.5.1**
 
-1. Entrez l'url du dépot https://github.com/YunoHost-Apps/grav_ynh
-2. Configurez l'application
-3. Cliquez sur installer
-4. Rendez-vous à l'url que vous avez configuré
+## Installation
 
-Depuis la ligne de commande:
+Use either YunoHost's admin panel or command line:
 
 ```sh
-sudo yunohost app install -l grav https://github.com/YunoHost-Apps/grav_ynh
+sudo yunohost app install --url grav https://github.com/YunoHost-Apps/grav_ynh
 ```
 
-Options:
-
-1. __Choisissez un domaine pour Grav__  
-Domaine pour votre installation (example.org, grav.example.org).
-
-2. __Choisissez un chemin pour Grav__  
-Chemin pour votre installation (/grav, /).
-
-3. __Est-ce une page publique ?__  
-Si cette application est publique, n'importe quelle personne pourra y accéder (example.org/grav).
-
-4. __Souhaitez-vous installer le plugin d'administration ?__  
-Grav possède un plugin d'administration optionnel, répondez `Yes` si vous souhaitez l'utiliser.
-
-5. __Souhaitez-vous rendre l'administration privée ?__  
-_Valide uniquement si le plugin d'administration est installé._  
-Protège l'administration derrière l'authentification Yunohost.
-
-6. __Souhaitez-vous créer un administrateur automatiquement ?__  
-_Valide uniquement si le plugin d'administration est installé et privé._  
-Si `Yes`, un compte administrateur sera automatiquement créé à votre première visite de votre instance Grav, et vous serez redirigé vers l'administration.
-Votre connexion à l'administration sera associée à votre connexion YunoHost.  
-Si `No`, à votre première visite de votre instance Grav, il vous sera proposé la création d'un nouvel utilisateur.
-
-
-## Versionning
-
-### Version 1.3.1 (22/07/17)
-
-- Update version
-
-### Version 1.1.1 (13/03/17)
-
-- Fix package_check [source](https://github.com/YunoHost/package_check)
-
-### Version 1.1.0 (02/03/17)
+```sh
+sudo yunohost app upgrade --url https://github.com/YunoHost-Apps/grav_ynh kanboard
+```
 
-- Fix Packaging [Source](https://forum.yunohost.org/t/news-about-app-management-and-packaging-in-yunohost-2-4/1379/1)
+# State of this package
 
-### Version 1.0.0 (11/11/15)
+* works fine:
+[x] install/remove/backup/remove/upgrade with x86_64
 
-- First version
+* to be confirmed:
+[ ] upgrade from old package to new package
```

Comment:
We could improve the code here and below, the checkboxes are displayed on one line.